src/cli.js

```diff
@@ -143,20 +143,27 @@ module.exports = function (inputArgs, cb) {
         return cli(inputArgs);
     }).then(function () {
         if (shouldCollectTelemetry && !isTelemetryCmd) {
-            telemetry.track(cmd, 'successful');
+            telemetry.track(cmd, subcommand, 'successful');
         }
         // call cb with error as arg if something failed
         cb(null);
     }).fail(function (err) {
         if (shouldCollectTelemetry && !isTelemetryCmd) {
-            telemetry.track(cmd, 'unsuccessful');
+            telemetry.track(cmd, subcommand, 'unsuccessful');
```

Comment:
Is there no need to check if subcommand is undefined? E.g. if the user runs `cordova platform`?